consumer.go

```diff
@@ -264,6 +285,10 @@ func (c *Consumer) rebalance() error {
 		return err
 	}
 
+	adds, rems := Diff(oldSubs, subs)
```

Comment:
I think we should a configuration option for that, something like `Config.Group.Return.RebalanceEvents`, defaulting to false, in line with sarama standard configuration